skepticoin/scripts/mine.py

```diff
@@ -23,83 +28,193 @@
 )
 
 
-def main():
-    parser = DefaultArgumentParser()
-    args = parser.parse_args()
-    configure_logging_from_args(args)
+def run_miner(args: Any, wallet_lock: synchronize.Lock, q: Queue, miner_id: int) -> None:
```

Comment:
`q` -> `queue`, here and elsewhere (bikeshedding, I know)

Reply:
I had to `import queue` earlier, but that import is gone now so using `queue` works again.